### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by bajajcodes**
- Cart page get products added to cart.
- Remove products from cart or move to wishlist.
- On single column layout cards container is having fixed height which makes cards stack to be scrollable in y-direction and cart section is placed at bottom.
- On multi column layout cards container auto-fill cards using display grid and place cart section on right of viewport.

https://user-images.githubusercontent.com/29247011/154842366-cf83aafc-d3b7-4dac-a4a7-9ab1e8726f67.mp4

### CODE REVIEW

File: css/products.css [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    background-color: var(--aside-beautify-color);
+    min-height: 100vh;
+}
+
+.aside-hamburger {
+    background-color: transparent;
+    border: 0;
+    font-size: var(--size-10);
+    cursor: pointer;
+    /* transition: transform 560ms; */
```
Comment (nareshpoonia) [inline css/products.css]: Remove comments

**Review by nareshpoonia — APPROVED** [review]
Code is clean. Just one comment where unused code is written & commented.
Good to Merge

File: css/products.css (hunk 1 above)
Comment (bajajcodes) [reply css/products.css]: I have implemented the suggested change.

**Review by Hardiegogo — COMMENTED** [review]
Code looks fine.
Just one small suggestion:
On hovering over the card, It takes up a large amount of space and clashes with other cards.
You can either try increasing the gap between cards or you can decrease the amount of space your card takes on hover.

**Pull request merged**